### PR TITLE
feat: impl text parser support encoding: false

### PR DESCRIPTION
**Pull request by killagu**
text parser support with no decoder,
to get buffer instead of string.

### CODE REVIEW

**Comment by killagu** [comment]
cc: @dead-horse

File: lib/text.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ module.exports = function(req, opts){
   var len = req.headers['content-length'];
   var encoding = req.headers['content-encoding'] || 'identity';
   if (len && encoding === 'identity') opts.length = ~~len;
-  opts.encoding = opts.encoding || 'utf8';
+  opts.encoding = typeof opts.encoding === 'undefined' ? 'utf8': opts.encoding;
```
Comment (dead-horse) [inline lib/text.js]: opts.encoding === undefiend ? 'utf8' : opts.encoding

Comment (killagu) [reply lib/text.js]: done

Comment (killagu) [reply lib/text.js]: @dead-horse done

**Pull request merged**